soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -2877,12 +2878,27 @@ namespace Settings {
     } else {
         GanonsTrialsCount.SetSelectedIndex(cvarSettings[RSK_TRIAL_COUNT]);
     }
+    SetDungeonTypes.SetSelectedIndex(false);
```

Comment:
What does this line do?

Reply:
I needed to disable the variable when you don't have selection mode enabled so I opted to have it set it off before it runs the checks for the MQ selection instead of duplicating the line twice in RO_MQ_DUNGEONS_NONE and RO_MQ_DUNGEONS_SET_NUMBER. If there is a better way of doing it let me know.